src/game.cpp

```diff
@@ -1843,12 +1847,19 @@ void game::add_npc_follower( const int &id )
 } ) ) {
         follower_ids.push_back( id );
     }
+    if( !std::any_of( u.follower_ids.begin(), u.follower_ids.end(), [id]( int i ) {
+    return i == id;
+} ) ) {
+        u.follower_ids.push_back( id );
+    }
 }
 
 void game::remove_npc_follower( const int &id )
 {
     follower_ids.erase( std::remove( follower_ids.begin(), follower_ids.end(), id ),
                         follower_ids.end() );
+    follower_ids.erase( std::remove( u.follower_ids.begin(), u.follower_ids.end(), id ),
```

Comment:
Possible cause of https://github.com/CleverRaven/Cataclysm-DDA/issues/29278.

I think it should be:

```suggestion
    u.follower_ids.erase( std::remove( u.follower_ids.begin(), u.follower_ids.end(), id ),
```